ui/angular.json

```diff
@@ -106,7 +106,8 @@
               "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
               "src/styles.scss"
             ],
-            "scripts": []
+            "scripts": [],
+            "codeCoverage": true
```

Comment:
Please also include `npm test` as part of the `make test` and `make test-coverage` targets in the `Makefile`

Reply:
added